File: coding-test/nodejs/src/practice/find-kimseobang-in-seoul.js
// @ts-check

/**
 *
 * @param {string[]} seoul
 * @returns {string}
 */
function solution(seoul) {
  return `김서방은 ${seoul.findIndex((x) => x === 'Kim')}에 있다`
}

const seoul = ['Jane', 'Kim']
console.log(solution(seoul))

File: practice-coding-test/java-kotlin/src/kr/co/programmers/java/practice/FindKimSeobangInSeoul.java
package kr.co.programmers.java.practice;

import java.util.Arrays;

public class FindKimSeobangInSeoul {

    public static void main(String[] args) {
        String[] seoul = {"Jane", "Kim"};
        System.out.println(new Solution().solution(seoul));
    }

    static class Solution {
        public String solution(String[] seoul) {
            //x에 김서방의 위치를 저장하세요.
            int x = Arrays.asList(seoul).indexOf("Kim");

            return "김서방은 "+ x + "에 있다";
        }
    }
}
